src/serac/physics/solid_mechanics.hpp

```diff
@@ -132,10 +132,12 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
             FiniteElementState::Options{
                 .order = order, .vector_dim = mesh_.Dimension(), .name = detail::addPrefix(name, "shape_displacement")},
             sidre_datacoll_id_)),
-        nodal_forces_(mesh_, displacement_.space(), "nodal_forces"),
+        reactions_(mesh_, displacement_.space(), "reactions"),
         shape_sensitivity_(mesh_, displacement_.space(), "shape_sensitivity"),
-        ode2_(displacement_.space().TrueVSize(), {.c0 = c0_, .c1 = c1_, .u = u_, .du_dt = du_dt_, .d2u_dt2 = previous_},
+        ode2_(displacement_.space().TrueVSize(),
+              {.time = ode_time_point_, .c0 = c0_, .c1 = c1_, .u = u_, .du_dt = du_dt_, .d2u_dt2 = previous_},
```

Comment:
Wouldn't all these member variables be uninitialized at this point? I realize this was an existing problem...